### ISSUE TITLE
Backend part of MERN app in server folder

### ISSUE THREAD

**Issue by Akhil-Sharma-26** [Akhil-Sharma-26/mikro-projects#1]
Add: 
- [x] connecting to database
- [x] login and signup
- [x] forgot password
- [x] verification

**Comment by 05-yuvraj-singh** [comment]
i can implement these features via mongo and express/node js . Do assign it to me if no one's working on it :)

**Comment by Akhil-Sharma-26** [comment]
> i can implement these features via mongo and express/node js . Do assign it to me if no one's working on it :)

Yeah, I will assign you this task. I have already implemented the logic of database connection, login and signup. Your task will be to try to implement the logic of forgot password and verification of user by email or something.

You should make a new folder inside server folder and then do your work there. `server/new_folder` .

For context, this is just a testing repository, I have made for just polishing my skills in different fields and to learn new concepts and use it like an archive, so not much updates or a proper project will be coming into this repo for the intergration of the above said tasks.

**Comment by 05-yuvraj-singh** [comment]
Yeah , i understood it. I am also here just to learn some basic implementation of mern stack . Forgot password would be easy i guess but verification via email would be a new thing to me . will keep updating . thanks :)

**Comment by Akhil-Sharma-26** [comment]
> Yeah , i understood it. I am also here just to learn some basic implementation of mern stack . Forgot password would be easy i guess but verification via email would be a new thing to me . will keep updating . thanks :)

Any Progress?
